### DISCUSSION TITLE
Ethernet esp32

### DISCUSSION THREAD

**Discussion by MattBoog** [tbnobody/OpenDTU#211] [Q&A] [answered]
I'd like to make use of an ESP32-ETH01, which is an ESP32-S1 with an ethernet port. Would that work? If so, would I have to change the code? I am specifically looking for ethernet, because I'd prefer not to use PoE.
The document for it can be found [here](https://www.tinytronics.nl/shop/index.php?route=product/product/get_file&file=3612/WT32-ETH01_manual.pdf)

**Comment by tbnobody** [comment, marked as answer]
There is no need to change the code. You only have to adjust the platformio.ini, copy e.g. the entry for the olimex poe device and adjust the pins for your needs.

**Reply by MattBoog** [reply]
Thank you!

**Comment by muibusan** [comment]
I don't use the Wifi module either. I use [Olimex ESP32-EVB](https://www.olimex.com/Products/IoT/ESP32/ESP32-EVB/open-source-hardware) (i.e. not the antenna version -EA!) with a LAN cable attached to its Ethernet port and it works flawlessly. [Olimex ESP-32-GATEWAY](https://www.olimex.com/Products/IoT/ESP32/ESP32-GATEWAY/open-source-hardware) should also work.
